hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSetProcessor.java

```diff
@@ -425,7 +425,7 @@ public StructType getExternalFileSchema(String storedAs, String location, boolea
                 fs = FileSystem.get(URI.create(location), conf);
                 String fileName = getFile(fs, location);
                 if (fileName != null) {
-                    temp = new Path(location, "temp");
```

Comment:
I think this could conflict with other systems, would it be better to create it under '_temporary' which seems more in line with what other systems do?